features/steps/fhir.py

```diff
@@ -144,7 +145,8 @@ def step_impl(context):
 @given('there is at least 1 entry')
 def step_impl(context):
     resource = context.response.json()
-    entries = resource.get('entry', [])
+    entries = [entry['resource'] for entry in resource.get('entry', [])
```

Comment:
I'm not checking entry type, because I'm pretty sure returning just a Condition result in an Observation query would raise an error somewhere else, but `entry.get('search', {}).get('mode', 'match')` can be read as "get the search property of the entry, defaulting to an empty object if not found, then get the mode property of that, defaulting to match if not found". Should result in returning "match" if the property was left off (which =='s match) or the actual value if present.